app/src/main/java/com/taskruler/RetrofitClientInstance.kt

```diff
@@ -5,14 +5,14 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://wrytheruc.github.io/"
+    private const val BASE_URL = "https://wrytheruc.github.io/"
```

Comment:
This variable is known at compile-time
https://codersee.com/retrofit-with-kotlin-the-ultimate-guide/

https://www.baeldung.com/kotlin/const-var-and-val-keywords